### ISSUE TITLE
Dependency error upon attempted install.

### ISSUE THREAD

**Issue by brimwats** [OleVik/grav-theme-scholar#19]
I am updated to

PHP 7.3 (which I think satisfies the >=7.1.3 <7.4 constraint)
Grav v1.6.31
Admin v1.9.19

When I try to install the theme I get: "Dependency error is required in an older version than the one installed. This package must be updated. Please get in touch with its developer."

**Comment by flberger** [comment]
Got hit by the same error, no idea how to fix it.

Since I am very interested in Scholar's WCAG features, any chance of an update that fixes this, @OleVik ?

**Comment by OleVik** [comment]
Sorry, this took me longer to get to than I'd have wanted. I'll examine this shortly.

**Comment by OleVik** [comment]
This should be resolved with v[3.0.0-beta.1](https://github.com/OleVik/grav-theme-scholar/releases/tag/3.0.0-beta.1). The error is in fact upstream, as the GPM does not do version-resolution. The major-version 3 will be loosening restrictions on interoperable versions, as Grav Core 1.7 for a long time was too unstable to work with. It's now settling, and work on compatibility can commence.

As version 3 stabilizes, there's also improvements to the WCAG-features planned, some optimizations are already in place.

**Comment by brimwats** [comment]
Great, thank you! Should I wait until I try installing? It looks like the latest grav version I have access to is 1.79

**Comment by OleVik** [comment]
It should be fine to install with any version bigger than Grav Core 1.7.0, so the current 1.7.9 works.

**Comment by brimwats** [comment]
Here's the error currently

![image](https://user-images.githubusercontent.com/49103046/111911552-4f11ff00-8a23-11eb-943d-d1a4e98ee40b.png)

**Comment by OleVik** [comment]
And this was when installing v3.0.0-beta.1 of Scholar? It's not on the stable channel by the way.

**Comment by brimwats** [comment]
> And this was when installing v3.0.0-beta.1 of Scholar? It's not on the stable channel by the way.

Ah yes, thanks! I tried it today. When I activate it I get a failure

```
        );
        $data = $YamlFile->load();
        if (isset($data['default'])) {
            $this->grav['config']->set('theme.schema.default', $data['default']);
        }
        if (isset($data['types'])) {
            foreach ($data['types'] as $schema => $data) {
                $this->grav['config']->set('theme.schema.types.' . $schema, $data);
            }
        }
        foreach ($this->config->get('themes.scholar.components') as $component) {
            $target = Utilities::fileFinder(
                'schema.yaml',
                [
                    'theme://components/' . $component,
                    'user://themes/scholar/components/' . $component
                ]
            );
            $file = $locator->findResource(
                $target,
                true,
                true
            );
            if (file_exists($file)) {
                $YamlFile = new YamlFile(
                    $file,
                    $formatter
                );
                foreach ($YamlFile->load() as $schema => $data) {
                    $this->grav['config']->set('theme.schema.types.' . $schema, $data);
                }
Arguments
"Invalid argument supplied for foreach()"


```

i actually get errors on all of these—let me know if I can share anything to help:

**Comment by brimwats** [comment]
![image](https://user-images.githubusercontent.com/49103046/112556178-d98d9200-8d86-11eb-9244-545cb3753c27.png)

**Comment by OleVik** [comment]
That appears to point to https://github.com/OleVik/grav-theme-scholar/blob/master/scholar.php#L426, which relies on the theme-configuration. Could you share your current /user/config/themes/scholar.yaml?

**Comment by OleVik** [comment]
[v2.0.1 of the skeleton package](https://github.com/OleVik/grav-skeleton-scholar) is updated to Core 1.7.10 and Admin 1.10.10 by the way, with an aligned ecosystem of extensions that have installed without problems in testing.

**Comment by brimwats** [comment]
thanks so much for the update! I've been meaning to get into grav and test things out but end of term grading has hit hard! I'm aiming to try everything again later this week if it's not a burden on your patience! 🙂

**Comment by brimwats** [comment]
Okay, here's what I did, with grav on Reclaim Hosting. 

1. deleted the current grav installation
2. reinstalled grav—relclaim does not have scholar as a skeletion option so I installed the default + admin.
3. downloaded version 2.0.1 as a zip: https://github.com/OleVik/grav-skeleton-scholar/releases/tag/2.0.1
4. using filezilla, uploaded the zip to the subdirectory 
5. unpacked the zip
6. refreshed the site
7. it works!

thanks so much! Cant wait to play with it :)
